modules/core/shared/src/main/scala/gem/math/Angle.scala

```diff
@@ -227,17 +239,91 @@ object Angle extends AngleOptics {
 
 }
 
-trait AngleOptics { this: Angle.type =>
+trait AngleOptics extends OpticsHelpers { this: Angle.type =>
+
+  /**
+   * Microarcseconds, in [0, 360°).
+   * @group Optics
+   */
+  lazy val microarcseconds: SplitMono[Angle, Long] =
+    SplitMono(_.toMicroarcseconds, Angle.fromMicroarcseconds)
+
+  /**
+   * Signed microarcseconds, in [-180°, 180°).
+   * @group Optics
+   */
+  lazy val signedMicroarcseconds: SplitMono[Angle, Long] = {
+    lazy val µas360 = Angle.Angle180.toMicroarcseconds * 2L
```

Comment:
Here I guess something has to be `lazy` since `Angle` extends this trait and hasn't yet defined `Angle180`.